Source/WebKit/UIProcess/WebAuthentication/fido/FidoService.cpp

```diff
@@ -71,13 +73,14 @@ void FidoService::continueAfterGetInfo(WeakPtr<CtapDriver>&& weakDriver, Vector<
     if (!driver || !observer() || response.isEmpty())
         return;
 
+    CTAP_RELEASE_LOG("Got response from getInfo: %s", base64EncodeToString(response).utf8().data());
```

Comment:
Not that all these `%s` in release logging will print nothing on customer builds. May be intentional but I thought I would pointer it out.